src/views/Profile/ProfileCreation/TeamSelection.tsx

```diff
@@ -73,25 +42,26 @@ const Team: React.FC = () => {
               'There’s currently no big difference between teams, and no benefit of joining one team over another for now. So pick whichever one you like!',
             )}
           </Text>
-          {teams.length === 0 && times(3).map((key) => <Skeleton key={key} height="80px" mb="16px" />)}
-          {teams.map((team, index) => {
-            // Team indices start at 1
-            const teamId = index + 1
+          {!teams && times(3).map((key) => <Skeleton key={key} height="80px" mb="16px" />)}
```

Comment:
This is was the error we were talking about. Before it was an empty array, now the initial value `null`.

Reply:
This 3 deserve its own variable , probably in a /config file

Reply:
Ah yes, you mean the teams list.